specifyweb/frontend/js_src/lib/components/Formatters/Definitions.tsx

```diff
@@ -124,9 +124,9 @@ function ConditionalFormatter({
 
   return (
     <div
-      className={`flex
-        ${isExpanded || !hasCondition ? 'flex-col' : 'items-center'}
-        ${isExpanded ? 'gap-2' : ''}
+      className={`flex gap-2 pt-2
+        ${isExpanded || !hasCondition ? 'flex-col' : ' items-center pb-2'}
+        ${isExpanded ? '' : ''}
```

Comment:
this line should be removed now that you have empty string in each branch

Reply:
Fixed in https://github.com/specify/specify7/pull/4631/commits/a6835234a9eb538b0b9804213e150d5f6d207d44